### PR TITLE
feat: implemented new sidebar

### LINKED ISSUE #901: Implement new sidebar

### PR DESCRIPTION
**Pull request by birtony**
Fixes #901

Implemented desktop version of the new sidebar. Focus states and mobile version will be added in follow-up PRs.

image
image

Also note that the old dashboard layout (including the old sidebar) are kept intact:
image

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

**Comment by talwinder50** [comment]
@birtony  do you think we should any availble icon for did management  ?

**Review by rolsonquadras — APPROVED** [review]

**Comment by birtony** [comment]
> @birtony do you think we should any availble icon for did management ?

As discussed during our sprint planning meeting, we will just link `Settings` to `DID Management` for now

**Comment by sudeshrshetty** [comment]
> > @birtony do you think we should any availble icon for did management ?
> 
> As discussed during our sprint planning meeting, we will just link `Settings` to `DID Management` for now

Those are not just "DID Management" anymore, they have user-preferences settings too. All those should be part of dev settings. Wallet settings is for regular wallet users who don't know the meanings of these digital identifiers and settings.

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/wallet-web/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "@vue/cli-plugin-babel": "^4.1.0",
     "@vue/cli-plugin-eslint": "^5.0.0-beta.3",
     "@vue/cli-plugin-router": "^4.5.13",
-    "@vue/cli-service": "^4.1.0",
+    "@vue/cli-service": "^4.5.13",
```
Comment (talwinder50) [inline cmd/wallet-web/package.json:47]: "@vue/cli-service": "^4.5.13",  I tested this i local, this change is breaking the test. Roll back the cli version and hopefully test will pass.

Comment (birtony) [reply cmd/wallet-web/package.json]: This does not resolve the test issue

**Pull request merged**